### ISSUE TITLE
Training time

### ISSUE THREAD

**Issue by supremus** [oandrienko/fast-semantic-segmentation#3]
Hello,

I wonder, what's the training time, both for PSPNet baseline and ICNet. I am running PSPNet training at the moment, it looks like it takes ~3 days. Does that sound right?

**Comment by oandrienko** [comment]
Assuming you are training with gradient checkpointing and using the provided configs, it took about a week to train PSPNet (batch size 8) and the non-compressed ICNet (batch size 16) on a single 1080 Ti. This could definitely be reduced if you used a distributed training setup, which is supported by the training scripts. I also trained for 200K iterations, which you can try lowering. 

Hope this helps

**Comment by oandrienko** [comment]
As a small update, I recently got access to multi-GPU machine and had no problem training PSPNet50 in significantly less time (also about ~2/3 days). Will close this issue for now.
